libs/tdigest/src/test/java/org/elasticsearch/tdigest/BigCountTests.java

```diff
@@ -35,13 +37,25 @@ public void testBigMerge() {
         }
     }
 
+    /**
+     * Verify that, at a range of compression values, the size of the produced digest is not much larger than 10 times the compression
+     */
+    public void testCompression() {
```

Comment:
This is tangential to the main point of this PR, but part of the exploratory work and seems like a reasonable test in general.